File: src/main/java/com/mhl/mycompanybackend/pojo/UserRequest.java
package com.mhl.mycompanybackend.pojo;

public class UserRequest {
    private Long id;
    private String email;
    private String firstname;
    private String lastname;
    private String middlename;
    private String phone_number;

    public UserRequest() {
    }

    public UserRequest(Long id, String email,String firstname, String lastname, String middlename, String phoneNumber) {
        this.id = id;
        this.email = email;
        this.firstname = firstname;
        this.lastname = lastname;
        this.middlename = middlename;
        phone_number = phoneNumber;
    }

    public Long getId() {
        return id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getMiddlename() {
        return middlename;
    }

    public void setMiddlename(String middlename) {
        this.middlename = middlename;
    }

    public String getPhone_number() {
        return phone_number;
    }

    public void setPhone_number(String phone_number) {
        this.phone_number = phone_number;
    }

}

File: src/main/java/com/mhl/mycompanybackend/pojo/UserRequest.kt
package com.mhl.mycompanybackend.pojo

class UserRequest {
    var id: Long? = null
        private set
    @JvmField
    var email: String? = null
    @JvmField
    var firstname: String? = null
    @JvmField
    var lastname: String? = null
    @JvmField
    var middlename: String? = null
    @JvmField
    var phone_number: String? = null

    constructor()
    constructor(id: Long?, email: String?, firstname: String?, lastname: String?, middlename: String?, phoneNumber: String?) {
        this.id = id
        this.email = email
        this.firstname = firstname
        this.lastname = lastname
        this.middlename = middlename
        phone_number = phoneNumber
    }
}
